### PR TITLE
fix(client): rfq expiring before timer hits zero(ARTP-903)

### PR DESCRIPTION
**Pull request by zhouyuhe**
now RFQ not expiring before timer hits zero

### CODE REVIEW

File: src/client/src/apps/MainRoute/widgets/spotTile/epics/rfqEpics.ts [hunk 1]
```diff
@@ -97,28 +97,26 @@ export const rfqRequestEpic: ApplicationEpic<{}> = (action$, state$) =>
...
   action$.pipe(
     ofType<Action, RfqReceivedActionType>(TILE_ACTION_TYPES.RFQ_RECEIVED),
     mergeMap(action => {
-      const {currencyPair} = action.payload
+      const { currencyPair } = action.payload
       const cancel$ = action$.pipe(
         ofType<Action, RfqReceivedTimerCancellableType>(
           TILE_ACTION_TYPES.RFQ_RESET,
           TILE_ACTION_TYPES.RFQ_CANCEL,
+          TILE_ACTION_TYPES.RFQ_REJECT,
```
Comment (mozmorris) [inline src/client/src/apps/MainRoute/widgets/spotTile/epics/rfqEpics.ts]: Nice 👍

File: src/client/src/apps/MainRoute/widgets/spotTile/epics/rfqEpics.test.ts [hunk 2]
```diff
@@ -216,7 +216,7 @@ describe('rfqEpics', () => {
         const testScheduler = new MockScheduler()
 
         testScheduler.run(({ cold, expectObservable }) => {
-          const actionLifetime = '-a 9999ms b 999ms   -|'
+          const actionLifetime = '-a 9999ms e 999ms   -|'
```
Comment (mozmorris) [inline src/client/src/apps/MainRoute/widgets/spotTile/epics/rfqEpics.test.ts]: Out on interest, does this test still pass if you'd left it a `b` ?

Comment (zhouyuhe) [reply src/client/src/apps/MainRoute/widgets/spotTile/epics/rfqEpics.test.ts]: no, I tried `b` before, it failed.
Then I go through this `epicTest`, and think it should change to `e`

Comment (mozmorris) [reply src/client/src/apps/MainRoute/widgets/spotTile/epics/rfqEpics.test.ts]: Great 👍

**Pull request merged**